README.md

```diff
@@ -43,14 +43,15 @@ Driver support depends on the device you are using shimboot on. The `patch_rootf
 | [`dedede`](https://chrome100.dev/board/dedede)   | yes               | yes  | no       | yes       | yes         | yes      | yes       | yes      |
 | [`octopus`](https://chrome100.dev/board/octopus) | yes               | yes  | yes      | yes       | yes         | yes      | yes       | yes      |
 | [`nissa`](https://chrome100.dev/board/nissa)     | yes               | yes  | no       | yes       | yes         | yes      | yes       | yes      |
-| [`reks`](https://chrome100.dev/board/reks)       | no<sup>[1]</sup> | yes  | untested | untested  | untested    | no       | untested  | untested |
-| [`kefka`](https://chrome100.dev/board/kefka)     | no<sup>[1]</sup> | yes  | yes      | yes       | untested    | no       | untested  | untested |
+| [`reks`](https://chrome100.dev/board/reks)       | no<sup>[1]</sup>  | yes  | untested | untested  | untested    | no       | untested  | untested |
+| [`kefka`](https://chrome100.dev/board/kefka)     | no<sup>[1]</sup>  | yes  | yes      | yes       | untested    | no       | untested  | untested |
 | [`zork`](https://chrome100.dev/board/zork)       | yes               | yes  | no       | untested  | yes         | yes      | yes       | yes      |
-| [`grunt`](https://chrome100.dev/board/grunt)     | yes               | yes  | no       | yes       | yes         | yes      | yes       | yes      |
+| [`grunt`](https://chrome100.dev/board/grunt)     | yes<sup>[2]</sup> | yes  | no       | yes       | yes         | yes      | yes       | yes      |
 | [`jacuzzi`](https://chrome100.dev/board/jacuzzi) | yes               | yes  | no       | yes       | untested    | no       | no        | yes      |
 | [`corsola`](https://chrome100.dev/board/corsola) | yes               | yes  | untested | untested  | untested    | untested | untested  | untested |
 
 <sup>1. The kernel is too old.</sup>
+<sup>2. The frames will not draw correctly on BARLA. (possible LightDM issue?)</sup>
```

Comment:
Can you be more specific about "frames will not draw correctly".

Reply:
Ok so, it will only draw the newest frame whenever you switch to a VT (e.g: CTRL+ALT+F1) and then back to the LightDM VT (CTRL+ALT+F7), but you need to do this for *every* frame.